Sources/Sentry/SentryClient.m

```diff
@@ -221,7 +221,15 @@ - (SentryEvent *)buildErrorEvent:(NSError *)error
 {
     SentryEvent *event = [[SentryEvent alloc] initWithError:error];
 
-    NSString *exceptionValue = [NSString stringWithFormat:@"Code: %ld", (long)error.code];
+    NSString *exceptionValue;
+
+    // If the error has a debug description, use that.
+    NSString *customExceptionValue = [[error userInfo] valueForKey:NSDebugDescriptionErrorKey];
+    if (customExceptionValue != nil) {
+        exceptionValue = customExceptionValue;
```

Comment:
`m`: Can we please add the error code at the end, as pointed out by you in https://github.com/getsentry/sentry-cocoa/pull/1496#discussion_r960753920